var/spack/repos/builtin/packages/kokkos-nvcc-wrapper/package.py

```diff
@@ -11,11 +11,11 @@ class KokkosNvccWrapper(CMakePackage):
        'full' C++ compiler that accepts all flags"""
 
     homepage = "https://github.com/kokkos/kokkos"
-    git = "https://github.com/kokkos/nvcc_wrapper.git"
+    git = "https://github.com/simonpintarelli/nvcc_wrapper.git"
 
-    version('3.1', url='https://github.com/kokkos/nvcc_wrapper/archive/3.1.tar.gz',
-            sha256="5171530763bb2845aa70651f3a8e5dc95b6eed76379323fde3205f97ed28ec8c",
-            default=True)
+    # version('3.1', url='https://github.com/kokkos/nvcc_wrapper/archive/3.1.tar.gz',
+    #         sha256="5171530763bb2845aa70651f3a8e5dc95b6eed76379323fde3205f97ed28ec8c",
+    #         default=True)
```

Comment:
Was this change intended to be part of this PR?

Reply:
Oh, very sorry, this was not intended to slip into this PR. I updated kokkos_nvcc wrapper for my own. Should have used a feature branch ...